etc/config/llvm_mir.amazon.properties

```diff
@@ -0,0 +1,65 @@
+compilers=&llc
```

Comment:
This might generate some issues with identically named groups and compilers. Can you please prepend some common prefix to all of them please? Maybe `mirllc`, `mirllc33`,... And so on :)